devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -314,15 +314,16 @@ def zwaveEvent(DoorLockOperationReport cmd) {
 	}
 	if (generatesDoorLockOperationReportBeforeAlarmReport()) {
 		// we're expecting lock events to come after notification events, but for specific yale locks they come out of order
-		runIn(3, "delayLockEvent", [data: [map: map, result: result]])
+		runIn(3, "delayLockEvent", [data: [map: map]])
         return [:]
 	} else {
 		return result ? [createEvent(map), *result] : createEvent(map)
 	}
 }
 
 def delayLockEvent(data) {
-	sendEvent(data.result ? [createEvent(data.map), *data.result] : createEvent(data.map))
+	log.debug "Sending cached lock operation: $data.map"
+	sendEvent(data.map)
```

Comment:
Cool. I missed to check that `sendEvent` takes a Map and not a list. However, with this we will miss to send the `HubActions` added to `result` list in `zwaveEvent(DoorLockOperationReport cmd)`. Are we sure that this will not break anything?

Reply:
Those hubActions are primarily for fixing an association issue that came up with first-generation Schlage locks. I expect it won't be a problem to skip them for recent Yale models.